lms/djangoapps/shoppingcart/processors/CyberSource2.py

```diff
@@ -123,7 +123,8 @@ def processor_hash(value):
     """
     secret_key = get_processor_config().get('SECRET_KEY', '')
     hash_obj = hmac.new(secret_key.encode('utf-8'), value.encode('utf-8'), sha256)
-    return binascii.b2a_base64(hash_obj.digest())[:-1]  # last character is a '\n', which we don't want
+    signature = binascii.b2a_base64(hash_obj.digest())[:-1]  # last character is a '\n', which we don't want
+    return signature.decode('utf-8')
```

Comment:
why not 
binascii.b2a_base64(hash_obj.digest())[:-1].decode('utf-8')

Reply:
it's more readable